cmd/root.go

```diff
@@ -118,6 +118,33 @@ func configLocations() []string {
 	}
 }
 
+var commonSocketPaths = []string{
```

Comment:
@jpdrsn [claims](https://github.com/nektos/act/issues/1658#issuecomment-1517723861) `~/.rd/docker.sock` is also one of the possible locations.